hugie/config.py

```diff
@@ -0,0 +1,113 @@
+import srsly
+import typer
+
+from hugie.models import InferenceEndpointConfig
+
+app = typer.Typer()
+
+
+@app.command()
+def modify(
+    path: str,
+    accountId: str = typer.Option(
+        None, help="ID of the account (for private endpoints)"
+    ),
+    name: str = typer.Option(None, help="Name of the endpoint"),
+    type: str = typer.Option(
+        None, help="Type of endpoint, one of ['public', 'protected', 'private']"
+    ),
+    accelerator: str = typer.Option(
+        "None", help="Accelerator to use. One of ['CPU','GPU']"
+    ),
+    instanceType: str = typer.Option(None),
+    instanceSize: str = typer.Option(None),
+    minReplica: int = typer.Option(None, help="Minimum number of replicas"),
+    maxReplica: int = typer.Option(None, help="Maximum number of replicas"),
+    framework: str = typer.Option("huggingface", help="Framework to use"),
+    image: str = typer.Option(
+        None,
+        help="Image to use when deploying model endppint. Must be string representing a valid JSON, e.g. '{'huggingface': {}}'",
+    ),
+    repository: str = typer.Option(None, help="Name of the hf model repository"),
+    revision: str = typer.Option(None, help="Revision of the hf model repository"),
+    task: str = typer.Option(None, help="Task of the model"),
+    vendor: str = typer.Option(None, help="Vendor to use. One of ['aws','gcp']"),
+    region: str = typer.Option(None, help="Vendor specific region, e.g. 'us-east-1'"),
+    overwrite: bool = typer.Option(
+        False, "--overwrite", "-o", help="Overwrite inpiut file with updated config"
+    ),
+):
+    """
+    Modify an existing endpoint config file
+    """
+
+    config = InferenceEndpointConfig.from_json(path)
```

Comment:
does it work with a path that does not exist?

Reply:
No, it should fail. I will add tests and error handling for this.

Reply:
Error handling already covered by `load_json()`. In order to create the file (if it doesn't exist), we would need to have _all_ the other arguments set, otherwise the file would not be properly specified. I would leave this for the current PR and let's discuss about the next one in relation to #54.

Noe that I was also considering adding a `hugie config create` to create a skeleton config file. Perhaps these two will be related.